references/common/common.go

```diff
@@ -35,3 +39,22 @@ func BuildRun(ctx context.Context, app *api.Application, client client.Client, n
 
 	return appfile.Run(ctx, client, o, nil)
 }
+
+// GetFilenameFromLocalOrRemote returns the filename of a local path or a URL.
+// It doesn't guarantee that the file or URL actually exists.
+func GetFilenameFromLocalOrRemote(path string) (string, error) {
```

Comment:
I think this function is very useful. How about moving it from `references/common` to `pkg/utils`?

Reply:
refactored